packages/lib/shim.ts

```diff
@@ -169,7 +168,9 @@ const shim = {
 			} catch (error) {
 				if (shim.fetchRequestCanBeRetried(error)) {
 					retryCount++;
-					if (retryCount > options.maxRetry) throw error;
+					if (retryCount > options.maxRetry) {
+						throw error;
+					}
 					await shim.msleep_(retryCount * 3000);
```

Comment:
I think all the changes in this file should be undone, shouldn't they? In particular the default value for maxRetry was removed.